app/build.gradle

```diff
@@ -263,6 +264,7 @@ dependencies {
     api 'com.google.firebase:firebase-core:16.0.6'
 
     implementation "androidx.legacy:legacy-support-v4:${androidSupportVersion}"
+    implementation 'com.android.support:cardview-v7:28.0.0'
```

Comment:
Please refer the androidX version of this library (or the material design library)